src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,28 @@
+.stopwatch {
+  @include square(300px);
+  position: relative;
+  background-image: url(../images/dial.svg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+
+  &__long-clip {
+    position: absolute;
+    left: 146px;
+    top: 45px;
```

Comment:
it is better to use `%` here, in case you'll ever need a better/smaller watch 😉

Reply:
done